charts/spire/charts/spire-agent/README.md

```diff
@@ -12,13 +12,17 @@ A Helm chart to install the SPIRE agent.
 |-----|------|---------|-------------|
 | bundleConfigMap | string | `"spire-bundle"` |  |
 | clusterName | string | `"example-cluster"` |  |
+| extraContainers | list | `[]` |  |
```

Comment:
Are these acting as sidecar containers? Is `extra` a standard nomenclature for a setup like this?

Reply:
Yes.

Extra for stuff is a pretty common pattern going way way back. For example:
https://github.com/helm/charts/blob/master/stable/fluent-bit/values.yaml#L202-L207